servicetalk-opentracing-inmemory-api/src/main/java/io/servicetalk/opentracing/inmemory/api/InMemorySpan.java

```diff
@@ -26,10 +26,35 @@
 /**
  * A span that allows reading values at runtime.
  */
-public interface InMemorySpan extends Span, InMemoryTraceState {
+public interface InMemorySpan extends Span {
     @Override
     InMemorySpanContext context();
 
+    /**
+     * The hex representation of the traceId.
+     * @return hex representation of the traceId.
+     */
+    String traceIdHex();
```

Comment:
if we are going to break APIs wdyt about keeping these methods in a single location:

- `*IdHex` -> `context().traceState()` (already on the `InMemoryTraceState` interface).
- `*isSampled` -> `context.isSampled()` (already on the `InMemorySpanContext` interface).

Reply:
hum, sounds good. I kinda liked the shortcut, but its have no other meaning, so yes getting rid of them.